### PR TITLE
[Detection] Output saliency map and feature vector for MPA Det

### PR DESCRIPTION
**Pull request by eugene123tw**

### CODE REVIEW

File: mpa/det/inferrer.py [hunk 1]
```diff
@@ -32,6 +34,8 @@ def run(self, model_cfg, model_ckpt, data_cfg, **kwargs):
         self._init_logger()
         mode = kwargs.get('mode', 'train')
         eval = kwargs.get('eval', False)
+        dump_features = kwargs.get('dump_features')
+        dump_saliency_map = kwargs.get('dump_saliency_map')
```
Comment (goodsong81) [inline mpa/det/inferrer.py]: Why don't you add the default value 'False' as above?

**Review by goodsong81 — CHANGES_REQUESTED** [review]
Thank you for the PR.
I'd like to ask a minor change.

File: mpa/det/inferrer.py (hunk 1 above)
Comment (eugene123tw) [reply mpa/det/inferrer.py]: Added. Thanks!

**Review by goodsong81 — APPROVED** [review]
LGTM!

**Pull request merged**